prow.sh

```diff
@@ -227,8 +227,8 @@ configvar CSI_PROW_E2E_IMPORT_PATH "k8s.io/kubernetes" "E2E package"
 # of the cluster. The alternative would have been to (cross-)compile csi-sanity
 # and install it inside the cluster, which is not necessarily easier.
 configvar CSI_PROW_SANITY_REPO https://github.com/kubernetes-csi/csi-test "csi-test repo"
-configvar CSI_PROW_SANITY_VERSION 5421d9f3c37be3b95b241b44a094a3db11bee789 "csi-test version" # latest master
-configvar CSI_PROW_SANITY_IMPORT_PATH github.com/kubernetes-csi/csi-test "csi-test package"
+configvar CSI_PROW_SANITY_VERSION af2d21f66c88d7b4956c17d8123cdc563ca0ba0a "csi-test version" # latest master
```

Comment:
A tag would be more readable and make the comment obsolete.

If I remember correctly, a hash was used only because there was no suitable release.

Reply:
Makes sense.

@fengzixu, can you use tag v4.0.2 instead of a hash?

Reply:
Done

Reply:
Can you apply this change temporarily on your PR in csi-driver-host-path and see if it passes CI?  Unit test will fail but other CI should pass if this works.

Reply:
Sure